### PR TITLE
Enable kuttl testing

### PR DESCRIPTION
**Pull request by chipzoller**
## Related Issue(s)

#546 

## Description

* Initializes kuttl tests with a TestSuite file
* Writes the first test starting in the `Argo` directory

### CODE REVIEW

File: argo/application-field-validation/01-install.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- crd.yaml
+- application-field-validation.yaml
```
Comment (realshuting) [inline argo/application-field-validation/01-install.yaml:5]: Where is this file?

Comment (realshuting) [reply argo/application-field-validation/01-install.yaml:5]: I didn't get it. Isn't this the file that needs to be created?

Comment (chipzoller) [reply argo/application-field-validation/01-install.yaml:5]: It already exists. Check the repo.

Comment (chipzoller) [reply argo/application-field-validation/01-install.yaml:5]: https://github.com/kyverno/policies/blob/main/argo/application-field-validation/application-field-validation.yaml

**Review by realshuting — APPROVED** [review]

File: argo/application-field-validation/01-install.yaml (hunk 1 above)
Comment (realshuting) [reply argo/application-field-validation/01-install.yaml:5]: My bad, I thought this was a new set of the kuttl test.

**Comment by chipzoller** [comment]
Merging this PR caused rendering to fail due to https://github.com/kyverno/website/issues/822. Sent PR #576 as a temporary fix until a solution can be found.

**Pull request merged**